crates/ruff_linter/src/rules/airflow/rules/moved_to_provider_in_3.rs

```diff
@@ -903,74 +897,86 @@ fn check_names_moved_to_provider(checker: &Checker, expr: &Expr, ranged: TextRan
         },
 
         // apache-airflow-providers-standard
-        ["airflow", "operators", "datetime"] => Replacement::ImportPathMoved{
+        ["airflow", "operators", "bash", ..] => Replacement::ImportPathMoved{
+            original_path: "airflow.operators.bash",
+            new_path: "airflow.providers.standard.operators.bash",
+            provider: "standard",
+            version: "0.0.1"
+        },
+        ["airflow", "operators", "bash_operator", ..] => Replacement::ImportPathMoved{
+            original_path: "airflow.operators.bash_operator",
+            new_path: "airflow.providers.standard.operators.bash",
+            provider: "standard",
+            version: "0.0.1"
+        },
```

Comment:
Should the `new_path` here be `bash_operator` (similar to the change above in `bash`) ?

Reply:
No, it should be `bash`. we move all these things to `airflow.providers.standard.operators.bash`